web/README.md

```diff
@@ -7,21 +7,14 @@ and it was bootstrapped with [Create React App](https://github.com/facebook/crea
 
```

Comment:
It still true that the application was bootstraped with Create React App... but not sure how relevant or confusing could that sentence be here.